server.js

```diff
@@ -114,7 +119,8 @@ let getRecipeInfo = function(recipeArrIds) {
         .then(function(recipeObjArr) {
             console.log(recipeObjArr);
             let newValues = recipeObjArr.map(recipe => {
-                newRecipes.push([recipe.title, recipe.spoonacularSourceUrl, recipe.image]);
+                newRecipes.push([recipe.readyInMinutes, recipe.title, recipe.spoonacularSourceUrl, recipe.image]);
```

Comment:
Could we move recipe.ReadyInMinutes to the end of the array?

Reply:
yes! of course.